web.test.tests/web.test.tests/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,121 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace web.test.tests.Pages
+{
+    class HistoryPage
+    {
+        private IWebDriver driver;
+
+        public static String ClearBtnId = "clear";
+
+        public HistoryPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public IWebElement Calculator => driver.FindElement(By.XPath("//div[text()='Calculator']"));
+        public IWebElement ClearBtn => driver.FindElement(By.Id(ClearBtnId));
+
+        public HistoryPage Clear()
+        {
+            ClearBtn.Submit();
+            return new HistoryPage(driver);
+        }
+
+        public DepositPage BackToDeposit()
+        {
+            Calculator.Click();
+            return new DepositPage(driver);
+        }
+
+        public int GetRowsNumber()
+        {
+            return driver.FindElements(By.XPath("//table[@id='history']/tr")).Count;
+        }
+
+        public IWebElement GetNRow(int row_number)
+        {
+            return driver.FindElement(By.XPath($"//table[@id='history']/tr[{row_number}]"));
+        }
+
+        public bool IsHistoryClean()
+        {
+            try
+            {
+                driver.FindElement(By.XPath($"//table[@id='history']/tr"));
+                return false;
+            } catch (NoSuchElementException e)
+            {
+                return true;
+            }
+        }
+
+        public String GetData(int row_number, int column_number)
+        {
+            return GetNRow(row_number).FindElement(By.XPath($"./td[{column_number}]")).Text;
+        }
+
+        public String GetAmount(int row_number)
+        {
+            return GetData(row_number, 1);
+        }
+
+        public String GetRate(int row_number)
+        {
+            return GetData(row_number, 2);
+        }
+
+        public String GetTerm(int row_number)
+        {
+            return GetData(row_number, 3);
+        }
+
+        public String GetYear(int row_number)
+        {
+            return GetData(row_number, 4);
+        }
+
+        public String GetDateFrom(int row_number)
+        {
+            return GetData(row_number, 5);
+        }
+
+        public String GetDateTo(int row_number)
+        {
+            return GetData(row_number, 6);
+        }
+
+        public String GetInterest(int row_number)
+        {
+            return GetData(row_number, 7);
+        }
+
+        public String GetIncome(int row_number)
+        {
+            return GetData(row_number, 8);
+        }
```

Comment:
I think mostly all methods above should be 'private', as you use them only inside this class.

Reply:
Done